spec/classes/sensu_package_spec.rb

```diff
@@ -30,12 +30,12 @@
     context 'setting version' do
       context 'without platform version suffix' do
         let(:params) { {
-          :version              => '0.9.10',
+          :version              => '0.29.0',
           :sensu_plugin_version => 'installed',
         } }
 
         it { should contain_package('sensu').with(
-          :ensure => '0.9.10.el7'
```

Comment:
This expected behavior is what was causing the issue. The actual expected behavior is if you don't specify el# then you don't use it. This may have been useful many releases ago when naming was different, though all releases now use the correct naming.

http://repositories.sensuapp.org/yum/7/x86_64/